### ISSUE TITLE
[Feature request] CraftTweaker support

### ISSUE THREAD

**Issue by jamesjimmy88** [TinyModularThings/IC2Classic#731]
Would it be possible to add basic CraftTweaker support by registering the mod's RecipeTypes? This would allow users to add and remove recipes in a much more convenient way compared to having to make custom datapacks.
For your convenience, this is how the Create devs do it:
https://github.com/Creators-of-Create/Create/blob/mc1.18/dev/src/main/java/com/simibubi/create/AllRecipeTypes.java#L82

**Comment by DarianLStephens** [comment]
Is there any way to customize recipes for the 1.7.10 version of the mod?
It seems that the plugin is only available for 1.12, and the .json system was also only implemented after that point.
Datapacks naturally didn't exist in 1.7.10, either.

I'm particularly interested in using the machines specifically to process some items.

**Comment by CrossVas** [comment]
You can use CraftTweaker and ModTweaker mods to add/change IC2 Recipes, as it was supported back then.
Unfortunately, the old wiki for CraftTweaker/ModTweaker 1.7.10 is gone, so no info can be found, but you can take a look at [ModTweaker GH page](https://github.com/jaredlll08/ModTweaker/tree/master-MC1.7.10/src/main/java/modtweaker2/mods/ic2c/mods), where you can find some info on supported machines.

Here is an example of Macerator recipe: 

```
import mods.ic2classic.Macerator;

Macerator.addRecipe(<minecraft:diamond_block>, <minecraft:dirt> * 64);
```

![image](https://user-images.githubusercontent.com/25727222/228706713-9be445da-9f7c-4edc-a850-507f6c5830bc.png)
